addon/components/simple-value.js

```diff
@@ -15,18 +15,22 @@ export default Component.extend({
   type: computed("value", function () {
     let v = this.get("value");
     return [true, false].includes(v)
-      ? "literal"
+      ? "boolean"
```

Comment:
Not sure why this was 'literal' before — 'boolean' is more appropriate.
Updated CSS too